react/tic-tac-toe/src/redux/store.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, compose, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { reducer as formReducer } from 'redux-form';
+import Game from './game/reducers';
```

Comment:
I'd recommend using lowercase when importing these files. Try to use Capitalized imports only on Components and Screens.